app/frontend/src/components/admin/ServiceClients.vue

```diff
@@ -22,7 +22,7 @@
         class="kc-table"
         :headers="headers"
         :items="serviceClients"
-        :items-per-page="10"
+        :items-per-page="7"
```

Comment:
This change doesn't seem to be working. I think it only takes 5, 10, 15...
10 is better IMO too

Reply:
ok. sounds good.

Reply:
i know you suggested making them alphabetical too. but i didnt get that done yet

Reply:
Let's leave the alphabetical stuff out for now.